recipes/r-alakazam/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = '0.2.7' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
```

Comment:
in bioconda we're not supporting windows; could you please remove these win-specific vars?